### ISSUE TITLE
Problem with conversion to Int16

### ISSUE THREAD

**Issue by ARdacious** [swharden/Csharp-Data-Visualization#4]
Hi, I was trying out your project, and found the input on my laptop to be erratic.

Clipping and overshooting of the samples. 

Checking your conversion from bytes to Int16 and then to double raised some suspicion. 
The data coming from my card seems to be signed, and due to some auto leveling feature, your conversion seems to work fine when i turn down the mics volume.

 When changing the conversion to use BitConversion:
`Int16 val = BitConverter.ToInt16(audioBytes, i * 2);`

The application started behaving as expected.

**Comment by swharden** [comment]
I’ll bet this bug exists because your microphone supports 32-bit recording while I developed this code with 16-bit recordings in mind.

I’ll take a closer look and modify the code to support all bit depths, then close this ticket when the issue is corrected.

Thanks for bringing this issue to my attention!

**Comment by ARdacious** [comment]
Yeah, I initially spent some time trying to see if it was in the samples / configuration. Nice that you picked it up so quickly!
